src/features/search/components/Search.tsx

```diff
@@ -50,9 +50,9 @@ export const Search: FC = () => {
     }
   };
 
-  const updateStock = (stock: StockProps) => dispatch(updateStockAction(stock));
+  const updateStock = (stock: Stock) => dispatch(updateStockAction(stock));
 
-  const selectOption = (data: StockProps) => {
+  const selectOption = (data: Stock) => {
```

Comment:
I think Alex pointed it out before but maybe make data a more relevant name

Reply:
Good spot, will change it